### PR TITLE
Add weight, height and body surface area in consultation form

### LINKED ISSUE #1678: ICU Module: missing fields: Weight, Height and Body Surface area

### PR DESCRIPTION
**Pull request by divyagar**
Closes #1678 

https://www.loom.com/share/86dc846948964e2c9d40282eb0658115

### CODE REVIEW

**Review by abhinandanarya06 — APPROVED** [review]

**Comment by nihal467** [comment]
testt approved

**Comment by bodhish** [comment]
@divyagar move it to the bottom (above advice maybe)

@nihal467 test in phone also

**Comment by samholics** [comment]
tested, failed

now weight and height fields moved to bottom 
also working on phone's interface.

but, where is it gonna reflect in the consultation history page?? 
there is no "Basic info" heading present in the consultation history page.

**Comment by nihal467** [comment]
test approved

**Comment by nihal467** [comment]
WhatsApp Image 2021-08-16 at 8 10 39 PM
WhatsApp Image 2021-08-16 at 8 12 09 PM

It should come where the green line appears

**Comment by nihal467** [comment]
@divyagar

File: src/Components/Facility/ConsultationDetails.tsx [hunk 1]
```diff
@@ -349,6 +348,52 @@ export const ConsultationDetails = (props: any) => {
...
+                          Height
+                        </h3>
+                        <div className="mt-2">
+                          {consultationData.height || "-"} cm
+                        </div>
+                      </div>
+                    </div>
+                  )}
+                {consultationData.weight != null &&
+                  consultationData.height != null &&
```
Comment (gigincg) [inline src/Components/Facility/ConsultationDetails.tsx:378]: There's no need to check x != null when you're checking x > 0

**Review by gigincg — APPROVED** [review]

**Pull request merged**